app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1581,7 +1581,7 @@ table li .account_details {
 
 .padder {
   height: 500px;
-  background: url("feast_hall_dressed_table_.jpg");
+  background: image_url("feast_hall_dressed_table_.jpg");
```

Comment:
Name of function `image_url` should be written in all lowercase letters with hyphens instead of underscores